src/amoc_api_scenario_handler.erl

```diff
@@ -81,8 +81,7 @@ resource_exists(Req, State = #state{resource = Resource}) ->
 -spec to_json(cowboy_req:req(), state()) -> 
     {jsx:json_text(), cowboy_req:req(), state()}.
 to_json(Req0, State = #state{resource = Resource}) ->
-    Status = amoc_controller:test_status(
-                     erlang:list_to_atom(Resource)),
+    Status = amoc_dist:test_status(erlang:list_to_atom(Resource)),
```

Comment:
Can we assume that in correct use case the target atom exists? If so, I suggest using function `list_to_existing_atom`. It's not safe to covert to atom from user's input. Atoms are not garbage collected so it's fairly easy to misuse this API and crash the app.

Moreover I've noticed that in first place there is conversion from binary to list. I think there is no need for that because there is `binary_to_atom/2` function which could be used here.